wiki/Contests/Skinning_Contest/4/en.md

```diff
@@ -0,0 +1,87 @@
+| ::{ flag="TR" }:: [Zeus-](https://osu.ppy.sh/users/5464437) | Contest planning |
+
+## Links
+
+- **[Contest Page](https://osu.ppy.sh/community/contests/178)**
+- [Announcement newspost](https://osu.ppy.sh/home/news/2023-05-08-skinning-contest-icons-of-an-era)
+- [Discussion Thread](https://osu.ppy.sh/community/forums/topics/1762288)
+- [Submission Thread](https://osu.ppy.sh/community/forums/topics/1762287)
+- [Discord server](https://discord.skinship.xyz)
+- [Official website](https://skinship.xyz)
+
+## Ruleset
+
+- Submissions must not contain inappropriate, malicious, or epileptic content, and should adhere to the [osu! community rules](/wiki/Rules).
+- All assets need to be created by yourself, or used with permission and proper credit to the authors. This includes, but is not limited to:
+  - fonts
+  - icons
+  - textures
+  - artworks (e.g. stock images or anime artwork)
+  - sounds
+- The majority of the in-game menu interface and at least two game modes have to be skinned. In other words, in addition to the gameplay elements, the following segments shouldn't be left default:
+  - ranking panel
+  - song selection
+  - mode selection
+  - mod icons
+  - pause and fail menu
+  - main menu (optional, but highly encouraged)
+- Submissions must be new creations — only skins posted within the submission period will be accepted.
+- Each submission must be accompanied by a forum thread in order to participate in the contest.
+- Submissions must be made in time. Latecomers will not be accepted.
+- Submissions must not be created as a part of, or be involved in a paid commission of any sort.
+- Submissions must be created individually. Teams/collaborations are not allowed.
+- Submissions must be in line with the given theme: "Icons of an Era".
+- The file size for your `.osk` submission must be below 100 MB.
+- Please submit a 16:9 image representing your skin, since this will be used as a cover on the contest page. This image must be at least 1280x720.
+
+**Failure to comply with the contest rules may lead to disqualification, or, in severe cases, exclusion from future contests.**
+
+## Theme: Icons of an Era {#theme}
+
+This iteration's theme, **Icons of an Era**, aims to spark the community's imagination by creating art around various iconic osu! beatmaps that were crafted and risen to glory throughout osu!'s lifetime, as well as the many either retired, official or community-made osu! mascots that have been introduced over the course of the past 15 years.
+
+Whether your entry is focused around pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), a group of beatmap backgrounds from a specific era in osu! history, or just a skin focused around objectively the best mascot that has ever grazed osu! (Taikonator 4000), you will need to make sure that the theme is prominent as anyone should be able to open your entry and almost immediately recognise what the inspiration behind your entry was.
```

Comment:
the wiki uses neutral writing and a passive voice, so this paragraph should be removed or rewritten

Reply:
Im open for suggestions as I simply copy pasted this from the newspost and have no better idea how to write it